.github/workflows/build.yml

```diff
@@ -12,4 +12,4 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - uses: gradle/wrapper-validation-action@v1
-      - run: ./gradlew build javadocJar checkMappings mapNamedJar --stacktrace
+      - run: ./gradlew checkLicenses build javadocJar checkMappings --stacktrace
```

Comment:
No, why remove `mapNamedJar` ?

Reply:
It is a dependency of `genFakeSource`, which leads to `javadoc` and `javadocJar`

Reply:
The proof that this is ran: https://github.com/FabricMC/yarn/runs/2212467447#step:5:181

Reply:
Ok cool, all good then 👍